NIPostsReload/Presentation/Scenes/NIPostFeedModule/ViewState/PostFeedType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PostFeedType.swift
+//  NIPostsReload
+//
+//  Created by Denys Niestierov on 29.12.2023.
+//
+
+import Foundation
+
+enum PostFeedType: CaseIterable {
+    case list
+    case grid
+    case gallery
+    
+    var title: String {
+        switch self {
+        case .list:
+            "List"
```

Comment:
IMO you should use the same style code everywhere. here you don’t write a return, but in the case of a layout you write